### PR TITLE
Support defining UCR with YAML

### PR DESCRIPTION
**Pull request by emord**
@proteusvacuum since you asked yesterday

### CODE REVIEW

File: corehq/apps/userreports/tests/utils.py [hunk 1]
```diff
@@ -31,9 +32,9 @@ def get_sample_report_config():
 
 def get_sample_data_source():
     folder = os.path.join(os.path.dirname(__file__), 'data', 'configs')
-    sample_file = os.path.join(folder, 'sample_data_source.json')
+    sample_file = os.path.join(folder, 'sample_data_source.yaml')
```
Comment (proteusvacuum) [inline corehq/apps/userreports/tests/utils.py]: I think you may have forgotten to commit this file?

**Review by proteusvacuum — APPROVED** [review]
sweet!
YAUC: Yet Another UCR Config

File: corehq/apps/userreports/tests/utils.py (hunk 1 above)
Comment (emord) [reply corehq/apps/userreports/tests/utils.py]: actually that should have stayed json. fixed

**Review by dannyroberts — APPROVED** [review]
I'm into it

**Pull request merged**